assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/index.js

```diff
@@ -0,0 +1,223 @@
+/**
```

Comment:
Moved the component to its own folder because it was getting too large, hence I decided to have each step contents in their own sub-components.